src/store.js

```diff
@@ -39,14 +40,23 @@ store.dispatch(setFilter(getTags(lessonContext, courseContext)));
 let initialMode = true;
 let initialLanguage = 'nb';
 let initialWelcomeBox = true;
+let initialPath = '/scratch/astrokatt/astrokatt';
+let initialCheckboxes = {};
 
 if (typeof localStorage !== 'undefined') {
   if (localStorage.isStudentMode) { initialMode = JSON.parse(localStorage.isStudentMode); }
   if (localStorage.language) { initialLanguage = localStorage.language; }
   if (localStorage.welcomeBox) { initialWelcomeBox = JSON.parse(localStorage.welcomeBox); }
+  for (let i in localStorage) {
+    if(i !== 'isStudentMode' && i !== 'language' && i !== 'welcomeBox') {
+      initialPath = i;
+      initialCheckboxes = JSON.parse(localStorage[i]);
```

Comment:
```
const LoadFromLocalStorage = (key, defaultValue) => {
  try {
    return JSON.parse(localStorage[key]);
  }
  catch (e) {
    return defaultValue;
  }
};
```
Denne fungerer ikke på language for eksempel. Siden nb må være string. 

Når det gjelder initialPath så var det mest for startknappen, men dette blir gjort om i #292 uansett, men kan fikse litt på det nå. Og ta detaljene i det andre issuet?